### ISSUE TITLE
[document enhancement] compare with Microsoft.EntityFrameworkCore.Sqlite?

### ISSUE THREAD

**Issue by LeiYangGH** [msallin/SQLiteCodeFirst#166]
The readme doesn't mention the initial motivation to create the project. I'm a curious is it because there's no sqlite code first at that time? From commit history I noticed the oldest file was created 7 years ago. Maybe there's no other EF sqlite adapters at that time? I also found the [stackoverflow answer](https://stackoverflow.com/a/29460684/1518100) refering this repo.

Now there's `Microsoft.EntityFrameworkCore.Sqlite`, with which one can create sqlite code first way.
While this repo is really excellent, people may feel hard choosing between the two. 
So still any advantage using `SQLite.CodeFirst`?

One of the possible difference might be:

- `SQLite.CodeFirst` is using `System.Data.SQLite`, while `Microsoft.EntityFrameworkCore.Sqlite` uses `Microsoft.Data.Sqlite`.

**Comment by msallin** [comment]
Back then there was full Framework and Entity Framework 6. EF 6 didn't (and still does not AFAIK) offer Code First for SQLite.
It is possible to use EF 6 also with .NET Core / .NET 5 and hence I did the necessary adjustments to support this.

Long story short:
If you use EF Core, there is a built-in (much better) option to use.
If you use EF 6 (for whatever reason) AFAIK there is no better option than using this library.

Maintaining this lib do not take much time (updating dependencies from time to time, answering few questions and merge some PRs) and I plan to continue as long as it is useful for somebody.

I edited my answer on Stack Overflow to state this and I'm also going to update the README of this repo.

Here we go:

- https://github.com/msallin/SQLiteCodeFirst/blob/master/README.md#project-status
- https://stackoverflow.com/questions/22174212/entity-framework-6-with-sqlite-3-code-first-wont-create-tables/29460684#29460684
